index.markdown

```diff
@@ -6,6 +6,8 @@ layout: default
 ---
 
 [Click here to join the team!](https://tinyurl.com/wrc-jointheteam)
+[Join us on Strava](https://www.strava.com/clubs/wrc-east-bay)
```

Comment:
Put a blank line between each of the links so that they appear on separate lines.